test/fixtures.py

```diff
@@ -231,3 +171,13 @@ def dummy_config(tmpdir):
         cnf.write(f)
 
     return conf
+
+
+@pytest.fixture(autouse=True)
+def clean_etc():
```

Comment:
This cleanup is not triggered by SCAutolib. It should probably be added to cleanup or cleanup_

Reply:
This is an automatically used fixture. Check [this](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request) documentation. Also note, that this fixture is not a part of SCAutolib functionality, but is a fixture for **library unite tests**.